### PR TITLE
Update make version

### PR DESCRIPTION
**Pull request by haroldsphinx**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -50,7 +51,7 @@ dev:
 .PHONY: version
 version:
 	@if [ ${RELEASE_VERSION} ]; then \
-	sed -i "s|^version =.*|version = '"${RELEASE_VERSION}"'|" node/Cargo.toml; \
+	sed -i "s|^version =.*|version = '"${CARGO_VERSION}"'|" Cargo.toml; \
```
Comment (seunlanlege) [inline Makefile:54]:
```suggestion
	sed -i "s|^version =.*|version = '"${RELEASE_VERSION}"'|" Cargo.toml; \
```

File: Makefile [hunk 2]
```diff
@@ -4,7 +4,8 @@ REPO=composablefi
 SERVICE_NAME=composable
 INSTALL_DIR=docker/
 IMAGE_URL:=${REPO}/${SERVICE_NAME}
-CARGO_VERSION:=$(shell git tag --sort=committerdate | grep -E '^v[0-9]' | tail -1 )
+RELEASE_VERSION:=$(shell git tag --sort=committerdate | grep -E '^v[0-9]' | tail -1)
```
Comment (seunlanlege) [inline Makefile:7]:
```suggestion
```

File: .github/workflows/docker-release.yml [hunk 3]
```diff
@@ -37,7 +36,7 @@ jobs:
   
     - name: Set env
       run: |
-        echo "RELEASE_VERSION=$(git tag --sort=committerdate | grep -E '^v[0-9]' | tail -1 | cut -c2- )" >> $GITHUB_ENV
+        echo "RELEASE_VERSION=$(git tag --sort=committerdate | grep -E '^v[0-9]' | tail -1)" >> $GITHUB_ENV
```
Comment (seunlanlege) [inline .github/workflows/docker-release.yml:39]:
```suggestion
        echo "RELEASE_VERSION=$(git tag --sort=committerdate | grep -E '^v[0-9]' | tail -1 | cut -c2- )" >> $GITHUB_ENV
```

**Review by seunlanlege — CHANGES_REQUESTED** [review]

File: Makefile (hunk 2 above)
Comment (haroldsphinx) [reply Makefile:7]: We don't need to remove this...the way the docker release it's set, the release version needs to be set in the makefile and it doesnt affect the way release-pipeline works

File: .github/workflows/docker-release.yml (hunk 3 above)
Comment (haroldsphinx) [reply .github/workflows/docker-release.yml:39]: Same applies here

**Pull request merged**